test/services/incoming_notification_service_test.rb

```diff
@@ -35,6 +35,10 @@ def setup
   class LockingTest < ActiveSupport::TestCase
     self.use_transactional_tests = false
 
+    def setup
+      skip "see https://github.com/rails/rails/pull/46553" if Rails.version <= Gem::Version.new('7.0.6')
```

Comment:
it is not in the beginning of the test itself, because of this block:
```
    ensure
      assert_nil fiber.resume
    end
```

Reply:
What if you use threads instead of fibers? wouldn't the test still be valid?

Reply:
Probably. But it will be more messy to synchronize. If you remember 3scale/porta#3403
So we think it is reasonable to wait until next rails bump to re-enable them

Reply:
Yes, just trying to avoid extra work to rewrite these tests completely.